infra/modules/kubevela-addons/definitions/nixbuildnet.cue

```diff
@@ -0,0 +1,80 @@
+parameter: {
+    secretName: string
```

Comment:
This is unused. How do we get the `token`?

Reply:
It's still a WIP, @shlevy ! Calm down! 😆 
I've to open a pull request to interact with Atlantis. Marking it as a draft.